cmd/gen-go-sample/testdata/sample.want

```diff
@@ -62,6 +63,8 @@ func main() {
 	arg0 := flag.Int64("arg0", 42, "")
 	arg1 := flag.String("arg1", "foobar", "")
 	flag.Parse()
-	sampleMyMethod(*arg0, *arg1)
+	if err := sampleMyMethod(*arg0, *arg1); err != nil {
```

Comment:
Do we want to port this to other languages? e.g., for java, apache cli throws `IOException`. Should we try-catch it in main method or just let main method throw?

Reply:
I think letting main method throw in Java is fine. The difference is that Go doesn't use exceptions (usually). So before this change, `sampleMyMethod` can error and we just silently ignored it.